python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -304,7 +301,8 @@ def copy(self, keep_going):
         return this_slice
 
     def __contains__(self, j):
-        return j.id in self.objects.keys()
+        allObjK = self.objects.keys()
+        return j.id in allObjK
```

Comment:
The code is exactly the same before and after the change. When doing:
```python
j.id in self.objects.keys()
```
it only calls `self.objects.keys()` once.